qiskit/schemas/ibmq_device_qobj_schema.json

```diff
@@ -0,0 +1,109 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "http://www.qiskit.org/schemas/ibm_backend_device_qobj_schema.json",
+    "description": "OpenQuantum quantum object data structure for running experiments. IBM Backend specific instructions for experimental devices. Must also validate against qobj_schema.json",
+    "version": "1.0.0",
+    "definitions": {        
+        "openqasm_instructions": {
+            "oneOf": [
+                { 
+                    "properties": {
+                        "name": {
+                            "enum": ["u3", "U", "id", "u1", "u0", "u2", "cx", "CX", "snapshot", "reset", "barrier", "bfunc", "copy", "measure"]
```

Comment:
So the idea is that this is a generic IBMQ device schema, but in reality we use real schemas coming from a call to `backend.schemas()`? Then for current devices, they will not have `reset`.